bff_positioning/visualization/plot_uncertainty.py

```diff
@@ -15,9 +15,14 @@
 import matplotlib
 matplotlib.use('agg')
 import matplotlib.pyplot as plt
+matplotlib.use('TkAgg')
+import matplotlib.pyplot as plt
 
+from mpl_toolkits.axes_grid1.inset_locator import zoomed_inset_axes
+from mpl_toolkits.axes_grid1.inset_locator import mark_inset
+import numpy as np
 
-N_SAMPLES = 20
+N_SAMPLES =8
```

Comment:
Space missing before `8` (when declaring variables, there should be a space before and after the `=` sign - https://www.python.org/dev/peps/pep-0008/#other-recommendations)